### DISCUSSION TITLE
2011/05/31/%E6%9C%80%E6%B8%85%E9%97%B2%E7%9A%84%E7%88%B8%E7%88%B8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#330] [Announcements]
# 2011/05/31/%E6%9C%80%E6%B8%85%E9%97%B2%E7%9A%84%E7%88%B8%E7%88%B8

刚有了孩子的同事们，看上去一个比一个憔悴，甚至还有人直接就跟我抱怨当爹的艰辛。相比之下，我这个爸爸当得还真是轻松啊。

https://blog.qizhen.xyz/2011/05/31/%E6%9C%80%E6%B8%85%E9%97%B2%E7%9A%84%E7%88%B8%E7%88%B8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

haiyingjia
2011年06月2日 02:41 编辑
“恶狠狠得朝我大吼大叫” 怎么可能! ? 那是你自己恶狠狠地理解偏差

回复
ruanqizhen
2011年06月2日 09:07 编辑
小宝都是欺软怕硬的，肯定是小宝一见你就像老鼠见了猫:) 所以你没见过小宝恶狠狠的样子呗。

回复
新岛望月
2011年06月2日 00:46 编辑
小豆豆招人喜爱! 祝福阮老师的幸福生活!! 我刚开始学LABVIEW编程, 一个小问题请教阮老师: 先NEW一个VI, 在其中设一个 START 和一个 EXIT 按钮, 点 START 时打开一个新的VI, 点 EXIT 时结束当前VI, 我想到要用CASE结构,可还没成功,麻烦阮老师给个例子或链接. 谢谢!

回复
ruanqizhen
2011年06月2日 09:04 编辑
你是打算通过程序打开关闭VI？
可以参考一下这个链接第六章中的范例：https://ruanqizhen.wordpress.com/2009/08/30/%E7%A4%BA%E4%BE%8B%E4%B8%8B%E8%BD%BD/

回复
carol_su
2011年06月1日 19:38 编辑
阮老师真幸福！祝福豆豆，祝阖家幸福！
我最近在忙着学labview,在看您写的书，我看的进度有点慢啊，我要加油了！

回复
